src/tools/dev/xml/GenerateCMake.h

```diff
@@ -308,7 +303,10 @@ class CMakeGeneratorPlugin : public Plugin
                     libs9.push_back(tmp2);
                 }
                 else
+                {
+                    tmp.append(vtkversion);
                     libs9.push_back(tmp);
+                }
             }
```

Comment:
I assume `libs9` and `libs8` variables were introduced to handle either VTK 8 or VTK 9. Any reason to get rid of the `9` designation in `libs9` now?